<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="notesLineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="descriptionLineEdit"/>
   </item>
   <item row="3" column="1">
    <widget class="QTimeEdit" name="timeEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Notes</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>115</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="titleLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDateEdit" name="dateEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>titleLineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>Form</receiver>
   <slot>titleEntered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>597</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>notesLineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>Form</receiver>
   <slot>notesEntered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>descriptionLineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>Form</receiver>
   <slot>descriptionEntered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>572</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeEdit</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>Form</receiver>
   <slot>timeEntered(QTime)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateEdit</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>Form</receiver>
   <slot>dateEntered(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>templateNameEntered()</slot>
  <slot>titleEntered()</slot>
  <slot>notesEntered()</slot>
  <slot>descriptionEntered()</slot>
  <slot>timeEntered(QTime)</slot>
  <slot>dateEntered(QDate)</slot>
 </slots>
</ui>
